x-pack/plugin/core/template-resources/src/main/resources/ecs@mappings.json

```
@@ -23,6 +23,30 @@
             "unmatch_mapping_type": "object"
           }
         },
+        {
+          "ecs_event_original_non_indexed_keyword": {
+            "mapping": {
+              "type": "keyword",
+              "index": false,
+              "doc_values": false
+            },
+            "path_match": [
+              "event.original"
+            ]
+          }
+        },
+        {
+          "ecs_x509_public_key_exponent_non_indexed_long": {
```

```suggestion
          "ecs_non_indexed_long": {
```

Reason is that we can eventually extend it with other fields. Also changing it later should not be a breaking change.